_test_common/pubspec.yaml

```
@@ -10,7 +10,7 @@ dependencies:
   build_config: any
   build_runner_core: any
   build_test: any
-  crypto: ^2.0.0
+  crypto: ^3.0.0
```

If this doesn't _need_ to expand we would probably prefer:
```suggestion
  crypto: ">=2.0.0 <4.0.0"
```

---

Done